warehouse/templates/manage/settings.html

```diff
@@ -51,7 +51,7 @@ <h3>Delete Project</h3>
       Deleting will irreversibly delete this project.
     {% endif %}
     </p>
-    {% set action = request.route_path('manage.project.delete_project', project_name=project.normalized_name) %}
-    {{ confirm_button("Delete project", "Project Name", project.normalized_name, action=action) }}
+    {% set action = request.route_path('manage.project.delete_project', project_name=project.name) %}
```

Comment:
We actually don't need to change this line, we should use the normalized name for generating the path.

Reply:
@di Done